src/appfl/comm/globus_compute/globus_compute_server_communicator.py

```diff
@@ -36,7 +41,21 @@ def __init__(
         logger: Optional[ServerAgentFileLogger] = None,
         **kwargs,
     ):
-        gcc = Client()
+        if "compute_token" in kwargs and "openid_token" in kwargs:
+            ComputeScopes = ComputeScopeBuilder()
+            compute_login_manager = AuthorizerLoginManager(
+                authorizers={
+                    ComputeScopes.resource_server: AccessTokenAuthorizer(kwargs["compute_token"]),
+                    AuthScopes.resource_server: AccessTokenAuthorizer(kwargs["openid_token"]),
+                }
+            )
+            compute_login_manager.ensure_logged_in()
+            gcc = Client(
+                login_manager=compute_login_manager, 
+                code_serialization_strategy=CombinedCode()
+            )
```

Comment:
Do we want to have another case where either `compute_token` or `openid_token` is provided to remind the user that both tokens should be provided?

Reply:
I added an assertion to check both tokens are provided if one of them is provided.